go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/air-verse/air
 
-go 1.23
```

Comment:
why do u change this?

Reply:
Because I forgot this PR was open. I use my fork regularly and update it as needed. 

I'm closing this PR for now, as there doesn't seem to be much demand for it. 

I'll reopen it if requested.